kikaha-modules/kikaha-rocker/source/kikaha/rocker/RockerResponse.java

```diff
@@ -32,15 +32,40 @@
     @NonNull
     Integer statusCode = 200;
 
-    public RockerResponse paramObject( final Object entity ) {
-        this.entity.paramObject( entity );
+    public RockerResponse objects( final Object entity ) {
+        this.entity.setObjects( entity );
         return this;
     }
 
     public RockerResponse templateName( final String templateName ) {
-        this.entity.templateName( templateName );
+        this.entity.setTemplateName( templateName );
         return this;
     }
+
+    @Override
```

Comment:
Hey, @jmilagroso ...
What about lombok's `@Getter` and `@Setter` annotations? It would make your code less verbose. ;)